pkg/controller/versions/strategy_v2_3.go

```diff
@@ -198,6 +200,41 @@ func (v *versionStrategyV2_3) validateAddons(spec *v2.ControlPlaneSpec, allError
 	}
 	return allErrors
 }
+func (v *versionStrategyV2_3) validateServiceMeshExtensionsRemoved(ctx context.Context, cl client.Client, smcp metav1.Object) error {
+	logger := logf.Log.WithName("smcp-validator-2.3")
+
+	serviceMeshExtensions := &apiv1.ServiceMeshExtensionList{}
+	if err := cl.List(ctx, serviceMeshExtensions); err != nil {
+		fmt.Println(err)
+		if !errors.IsNotFound(err) {
+			logger.Error(err, "failed to list ServiceMeshExtensions")
```

Comment:
it failed because there's no SME CR, so, it's expected and not an error?

Reply:
not sure what you mean? the test passes locally

Reply:
I mean: trying to list a SME and get a failure should not be logged as an error, should it? If there's no SME CRD in the cluster, for example, an error is expected and desirable, right?

Reply:
Also, there's a println hidden in there :D

Reply:
> I mean: trying to list a SME and get a failure should not be logged as an error, should it? If there's no SME CRD in the cluster, for example, an error is expected and desirable, right?
Hmm in that case you'd be right. We will have to still install the CRD though, as a user might install a pre-2.3 version - and we always install the same set of CRDs for all control plane versions. But I can check for that error


> Also, there's a println hidden in there :D
not anymore, I removed that :wink: it's just still visible here for some reason